manifests/bucketeer/charts/dex/templates/deployment.yaml

```diff
@@ -65,6 +65,7 @@ spec:
                   - "-c"
                   - "sleep 10;"
           args:
+            - dex
```

Comment:
dex requires the `dex` arg from v2.30.0.
https://github.com/dexidp/dex/commit/dd4a62e64562ef82c2eb0c4aee34680a6fa50b8c#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R70